src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -114,7 +114,7 @@ public function doAsk(OutputInterface $output, Question $question)
         $inputStream = $this->inputStream ?: STDIN;
         $autocomplete = $question->getAutocompleterValues();
 
-        if (null === $autocomplete || !$this->hasSttyAvailable()) {
+        if ((null === $autocomplete || self::OS_WIN == $this->getOSTerminal()) || !$this->hasSttyAvailable()) {
```

Comment:
Adding new const + public method is a new feature.
Here, it's not needed, just do as we do everywhere else: `'\\' === _DIRECTORY_SEPARATOR`

Reply:
or use a private method here instead of putting this in the base Helper class